sycl/doc/extensions/experimental/sycl_ext_oneapi_get_unsupported_platforms.asciidoc

```diff
@@ -0,0 +1,123 @@
+= sycl_ext_oneapi_get_unsupported_platforms
+
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+:dpcpp: pass:[DPC++]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes. This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+
+== Notice
+
+[%hardbreaks]
+Copyright (C) 2024 Intel Corporation. All rights reserved.
+
+Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks
+of The Khronos Group Inc. OpenCL(TM) is a trademark of Apple Inc. used by
+permission by Khronos.
+
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/intel/llvm/issues
+
+
+== Dependencies
+
+This extension is written against the SYCL 2020 revision 8 specification. All
+references below to the "core SYCL specification" or to section numbers in the
+SYCL specification refer to that revision.
+
+
+== Status
+
+This is an experimental extension specification, intended to provide early
+access to features and gather community feedback. Interfaces defined in this
+specification are implemented in {dpcpp}, but they are not finalized and may
+change incompatibly in future versions of {dpcpp} without prior notice.
+*Shipping software products should not rely on APIs defined in this
+specification.*
+
+
+== Overview
+
+The implementation of `sycl::platform` already allows querying for supported
+platforms - `static std::vector<platform> get_platforms()` - which returns all
+available SYCL platforms in the system (the resulting vector always contains a
+single SYCL host platform instance). This extension adds an API entry to list
```

Comment:
NIT: Is "host" a supported platform?
I thought it was deprecated long ago.

Reply:
It was, that's a comment that I've lifted from source code without thinking too much about it. I'll rephrase it and remove the original one as well.

Done in 413dd3904e5879f715e6b461634a9d1f25445346